kata-deploy/scripts/install-kata-containerd.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+echo "copying kata artifacts onto host"
+cp -R /tmp/kata/* /opt/kata/
+chmod +x /opt/kata/bin/*
+cp /opt/kata/configuration.toml /usr/share/defaults/kata-containers/configuration.toml
+
+## Configure containerd to use Kata:
+echo "create containerd configuration for Kata"
+mkdir -p /etc/containerd/
+
+cat << EOT | tee /etc/containerd/config.toml
```

Comment:
Can you verify if the file exists and print a warning of find a way to merge the configs?

Reply:
Also can you create a backup file?

Reply:
This is just a reference, and I know this file doesn't exist by default in containerd.  I added support for creating a backup.